src/tokens/base.ts

```diff
@@ -23,6 +23,7 @@ export enum CoinKey {
   G = 'G',
   IMX = 'IMX',
   KLAY = 'KLAY',
+  OKB = 'OKB',
```

Comment:
OKB is a native on xlayer?

Reply:
yes, confirmed by me and Martin: https://www.okx.com/zh-hans/web3/explorer/xlayer/tx/0x9908a00144768f411b3cd2e5a01d314af6c6a65bd9dbc58a445974e141785de4